api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Function could be reduced down to just</p><p> </p><p><code></code><code></p><p>for series in gclient.rank_series_by_source(selection):</p><p>  return gclient.get_data_series(**series)</p><p></code><code></code></p><p> </p><p>Related to our StopIteration conversation from the other day, this would naturally return None if the generator doesn't yield any series, instead of needing to catch that edge case with a try/except</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>thanks!</p>

Reply:
<p>yeah just edit of that last line</p><p><code>return gclient.get_data_series(**series)[0<span class="ql-cursor">﻿</span>]</code></p><p> </p>